pkg/utils/common.go

```diff
@@ -73,6 +73,11 @@ func SplitKeyValueStringToMap(str string, delimiter string) (map[string]string,
 	return kvMap, nil
 }
 
+// Context for bootstrapping a Fuji Node
+func GetFujiBoostrappingContext() (context.Context, context.CancelFunc) {
```

Comment:
nit: have GetBootstrappingContext(network) where you can return context for switch/case for each network.
also typo in `GetFujiBoost`

Reply:
or change GetANRContext to add network param there

Reply:
added a method to network